src/main/target/OMNIBUSF7/target.h

```diff
@@ -72,15 +68,15 @@
 //#define UART2_RX_PIN PA3
 
 // Assigned to shared output I2C2
-//#define USE_UART3
-//#define UART3_RX_PIN PB10
-//#define UART3_TX_PIN PB11
+#define USE_UART3
+#define UART3_RX_PIN PB11
+#define UART3_TX_PIN PB10
```

Comment:
i generally dont like tabbed whitespace b/c when one line gets too long every line needs a change, which obfuscates the logical difference in the git history, but... if that's the style we're requiring...